common/src/main/java/org/vivecraft/client/network/ClientNetworking.java

```diff
@@ -221,7 +224,7 @@ public static void handlePacket(CommonNetworkHelper.PacketDiscriminators packetI
         Minecraft mc = Minecraft.getInstance();
         switch (packetID) {
             case VERSION -> {
-                String s11 = buffer.readUtf(1024);
```

Comment:
so remove the length limit? can do that

Reply:
Looking at it again now, this may not matter. The default in vanilla is much higher (32767). As such it's probably fine to keep? My original comments was me wondering why we need such a big buffer, but it seems vanilla uses an even bigger one. If we know the size of the string (or know it's smaller then 1024), keeping this is the better option it seems. I am still wondering where these "magic" numbers come from.